src/variables/functions.ts

```diff
@@ -0,0 +1,23 @@
+// Fast way to get element by ID
+export const $ = ( id: string ) => document.getElementById( id )
+// Puts all html code into main div
+export const render = ( html: string ) => {
+    const MainDiv = $( "app" ) as HTMLElement
+    MainDiv.innerHTML = html
+}
+// Check if string in lower case includes another string in lower case
+export const bIstoLCaseIncludes = ( data: string, equalTo: string ) => {
```

Comment:
Don't use "b" as the first letter in the **function names**. This is commonly used for booleans, but not functions which return booleans.

When making a name for a function returning a boolean, use words like `does` or `should` etc.

E.g. in this case `lowerCaseDoesInclude()`